runtime/src/test/scala/tailcall/runtime/internal/CalibanGen.scala

```diff
@@ -0,0 +1,52 @@
+package tailcall.runtime.internal
+
+import caliban.{InputValue, ResponseValue, Value}
+import zio.test.Gen
+
+import java.math.BigInteger
+import java.util.Random
+
+object CalibanGen {
+  val probablePrime = BigInt(BigInteger.probablePrime(100, new Random(0x9e3779b1L)))
```

Comment:
Can we use a simpler generator for BigInt if available?

Reply:
Done.